### PR TITLE
0.32.4 Release Notes

### PR DESCRIPTION
**Pull request by lzdanski**
Includes CVE's listed in: 

- https://github.com/astronomer/issues/issues?q=is%3Aissue+label%3A0.32.4

could access PR, but not identify details about CVEs included (I don't have all the same permissions as @jwitz) in the following PRs:

- ap-registry https://github.com/astronomer/issues/issues/5772
- ap base https://github.com/astronomer/issues/issues/5829
- ap-prometheus https://github.com/astronomer/issues/issues/5773

See conversation for more info: https://astronomer.slack.com/archives/C04JELRC6J3/p1697564937125329

### CODE REVIEW

**Review by danielhoherd — APPROVED** [review]
Seems fine to me.

File: software/release-notes.md [hunk 1]
```diff
@@ -107,7 +107,32 @@ This feature is off by default. You can enable it by setting  `deployments.pgBou
...
+    - [CVE-2023-37788](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37788)
+    - [CVE-2023-37920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37920)
+    - [CVE-2023-38039](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38039)
+    - [CVE-2023-38325](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38325)
+    - [CVE-2023-38545](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38545)
+    - [CVE-2023-39417](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39417)
+    - [CVE-2023-41721](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-41721)
+    - [CVE-2023-44487](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-44487)
+    - [CVE-2023-45133](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-45133)
+    - [CVE-2023-48174](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-48174)
```
Comment (jwitz) [inline software/release-notes.md:135]:
```suggestion
    - [CVE-2023-48174](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-48174)
    - [CVE-2022-21698](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21698)
    - [CVE-2021-33194](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-33194)
    - [CVE-2021-38561](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-38561)
    - [CVE-2023-40577](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-40577)
```

File: software_versioned_docs/version-0.30/release-notes.md [hunk 2]
```diff
@@ -104,6 +104,32 @@ Astronomer Software now automatically scales the size of PGBouncer connection po
...
+    - [CVE-2023-37788](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37788)
+    - [CVE-2023-37920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37920)
+    - [CVE-2023-38039](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38039)
+    - [CVE-2023-38325](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38325)
+    - [CVE-2023-38545](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38545)
+    - [CVE-2023-39417](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39417)
+    - [CVE-2023-41721](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-41721)
+    - [CVE-2023-44487](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-44487)
+    - [CVE-2023-45133](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-45133)
+    - [CVE-2023-48174](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-48174)
```
Comment (jwitz) [inline software_versioned_docs/version-0.30/release-notes.md:131]:
```suggestion
    - [CVE-2023-48174](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-48174)
    - [CVE-2022-21698](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21698)
    - [CVE-2021-33194](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-33194)
    - [CVE-2021-38561](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-38561)
    - [CVE-2023-40577](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-40577)
```

File: software_versioned_docs/version-0.32/release-notes.md [hunk 3]
```diff
@@ -106,6 +106,32 @@ Astronomer Software now automatically scales the size of PGBouncer connection po
...
+    - [CVE-2023-37788](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37788)
+    - [CVE-2023-37920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37920)
+    - [CVE-2023-38039](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38039)
+    - [CVE-2023-38325](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38325)
+    - [CVE-2023-38545](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38545)
+    - [CVE-2023-39417](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39417)
+    - [CVE-2023-41721](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-41721)
+    - [CVE-2023-44487](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-44487)
+    - [CVE-2023-45133](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-45133)
+    - [CVE-2023-48174](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-48174)
```
Comment (jwitz) [inline software_versioned_docs/version-0.32/release-notes.md:133]:
```suggestion
    - [CVE-2023-48174](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-48174)
    - [CVE-2022-21698](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21698)
    - [CVE-2021-33194](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-33194)
    - [CVE-2021-38561](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-38561)
    - [CVE-2023-40577](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-40577)
```

**Review by jwitz — APPROVED** [review]
Looks good to me! Added suggestions for the CVEs I found in the linked PRs.

**Review by himabindu07 — COMMENTED** [review]
Looks Good to me

**Review by himabindu07 — APPROVED** [review]

**Comment by rishkarajgi** [comment]
@lzdanski I think this one got missed out - https://github.com/astronomer/issues/issues/3253

File: software/release-notes.md [hunk 4]
```diff
@@ -107,7 +107,36 @@ This feature is off by default. You can enable it by setting  `deployments.pgBou
...
+## 0.32.4
+
+Release date: October 19, 2023
+
+### Bug fixes
+
+- Fixed an issue where `ap-blackbox-exporter` did not respect global network policies.
+
+- Fixed the following vulnerabilities:
+    - [CVE-2023-2253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-2253)
```
Comment (rishkarajgi) [inline software/release-notes.md:120]: This one was dropped and hence can be omitted

File: software/release-notes.md [hunk 5]
```diff
@@ -107,7 +107,36 @@ This feature is off by default. You can enable it by setting  `deployments.pgBou
...
+    - [CVE-2023-37920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37920)
+    - [CVE-2023-38039](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38039)
+    - [CVE-2023-38325](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38325)
+    - [CVE-2023-38545](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38545)
+    - [CVE-2023-39417](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39417)
+    - [CVE-2023-41721](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-41721)
+    - [CVE-2023-44487](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-44487)
+    - [CVE-2023-45133](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-45133)
+    - [CVE-2023-48174](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-48174)
+    - [CVE-2022-21698](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21698)
```
Comment (rishkarajgi) [inline software/release-notes.md:136]: This one was dropped and hence can be omitted

**Comment by rishkarajgi** [comment]
3 issues were dropped from this release
- https://github.com/astronomer/issues/issues/5773
- https://github.com/astronomer/issues/issues/5775
- https://github.com/astronomer/issues/issues/5772

File: software/release-notes.md (hunk 5 above)
Comment (lzdanski) [inline software/release-notes.md:136]:
```suggestion
```

File: software/release-notes.md (hunk 4 above)
Comment (lzdanski) [inline software/release-notes.md:120]:
```suggestion
```

File: software_versioned_docs/version-0.30/release-notes.md [hunk 6]
```diff
@@ -104,6 +104,36 @@ Astronomer Software now automatically scales the size of PGBouncer connection po
...
+## 0.32.4
+
+Release date: October 19, 2023
+
+### Bug fixes
+
+- Fixed an issue where `ap-blackbox-exporter` did not respect global network policies.
+
+- Fixed the following vulnerabilities:
+    - [CVE-2023-2253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-2253)
```
Comment (lzdanski) [inline software_versioned_docs/version-0.30/release-notes.md:116]:
```suggestion
```

File: software_versioned_docs/version-0.32/release-notes.md [hunk 7]
```diff
@@ -106,6 +106,36 @@ Astronomer Software now automatically scales the size of PGBouncer connection po
...
+## 0.32.4
+
+Release date: October 19, 2023
+
+### Bug fixes
+
+- Fixed an issue where `ap-blackbox-exporter` did not respect global network policies.
+
+- Fixed the following vulnerabilities:
+    - [CVE-2023-2253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-2253)
```
Comment (lzdanski) [inline software_versioned_docs/version-0.32/release-notes.md:118]:
```suggestion
```

File: software_versioned_docs/version-0.32/release-notes.md [hunk 8]
```diff
@@ -106,6 +106,36 @@ Astronomer Software now automatically scales the size of PGBouncer connection po
...
+    - [CVE-2023-37920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-37920)
+    - [CVE-2023-38039](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38039)
+    - [CVE-2023-38325](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38325)
+    - [CVE-2023-38545](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-38545)
+    - [CVE-2023-39417](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-39417)
+    - [CVE-2023-41721](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-41721)
+    - [CVE-2023-44487](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-44487)
+    - [CVE-2023-45133](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-45133)
+    - [CVE-2023-48174](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-48174)
+    - [CVE-2022-21698](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-21698)
```
Comment (lzdanski) [inline software_versioned_docs/version-0.32/release-notes.md:134]:
```suggestion
```

**Review by rishkarajgi — APPROVED** [review]
lgtm

**Pull request merged**